README.md

```diff
@@ -181,6 +181,23 @@ Class names like `Options` and `Level` might be too common, if you get a conflic
     })
     ...
 
+#### Angular Universal
+
+To use it with Angular Universal add angular2-logger to App.module normaly and in node.module provide these custom options.   
+
+```javascript
+import { Options } from "angular2-logger/core";
+
+@NgModule({
+  bootstrap: [ AppComponent ],
+  imports: [ AppModule, ... ],
+  providers: [
+    { provide: Options, useValue: { global: false, store: false } }, // stop angular2-logger from defining global variables.
```

Comment:
@langley-agm , maybe you can elaborate more in what the confusion is ? The browser is the only one that renders the HTML code, that's true. But the javascript code which generates the HTML code runs both in the client and in the server when you are using Angular Universal. 
Regarding the change breaking existing functionality, I suppose that if "store" is false it doesn't really make sense to check the localStorage, or does it ? Why do you need to check the localStorage even though the "store" option is false ?